code/cgame/cg_draw.c

```diff
@@ -849,6 +851,59 @@ static void CG_DrawHoldableItem(float y) {
 			CG_DrawPic(640 - ICON_SIZE, y, ICON_SIZE, ICON_SIZE, cgs.media.noammoShader);
 		}
 
+		if (itemId == (HI_BAMBAM || HI_BOOMIES)) {
```

Comment:
this boils down to `if (itemId == 1)` - you meant `if (itemId == HI_BAMBAM || itemId == HI_BOOMIES)`